.github/workflows/build-test-deploy.yml

```diff
@@ -293,7 +293,22 @@ jobs:
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
           name: hyperspace-dali
           artifact: hyperspace-dali:latest
-      
+      - name: Publish hyperspace-picasso to docker hub
```

Comment:
may this work with gh job matrix?

Reply:
yes, this may
but matrix strategy work on level `jobs.<job_id>`.
so CI will run the [job](https://github.com/ComposableFi/composable/blob/main/.github/workflows/build-test-deploy.yml#L219-L295) wholly for every runtime, i.e. 3 times.

Reply:
I moved a building of Hyperspace docker image to a separate CI job.